src/main/java/org/nastya/service/UserClient/HeaderConstants.java

```diff
@@ -0,0 +1,5 @@
+package org.nastya.service.UserClient;
```

Comment:
Move to a different package (org.nastya.service.controller). http header constants belong to controllers layer. It is not business logic.